packages/clay-card/src/Card.tsx

```diff
@@ -35,78 +34,69 @@ interface IProps
 			HTMLAnchorElement | HTMLSpanElement | HTMLDivElement
 		> {}
 
-const ClayCard: React.FunctionComponent<IProps> & {
-	AspectRatio: typeof AspectRatio;
-	Body: typeof Body;
-	Caption: typeof Caption;
-	Description: typeof Description;
-	Group: typeof Group;
-	Row: typeof Row;
-} = ({
+const ClayCard: React.FunctionComponent<IProps> = ({
 	children,
 	className,
 	displayType,
-	horizontal,
-	href,
-	interactive,
-	onClick,
 	selectable = false,
 	...otherProps
-}: IProps) => {
+}) => {
 	const isCardType = {
 		file: displayType === 'file',
 		image: displayType === 'image',
 		user: displayType === 'user',
 	};
 
-	const TagHeaderName = href ? ClayLink : 'div';
-
-	const TagName = interactive ? 'span' : 'div';
-
 	return (
-		<Context.Provider value={{horizontal, interactive}}>
-			<TagHeaderName
+		<Context.Provider value={{horizontal: false, interactive: false}}>
+			<div
+				className={classNames(
+					className,
+					{
+						card: !selectable,
+						'file-card': isCardType.file,
+						'form-check-card form-check form-check-top-left': selectable,
+						'image-card': isCardType.image,
+						'user-card': isCardType.user,
+					},
+					'card-type-asset'
+				)}
 				{...otherProps}
-				className={classNames(className, {
-					card: !selectable,
-					'card-interactive card-interactive-primary card-type-template':
-						(horizontal && interactive) || interactive,
-					'card-type-asset':
-						isCardType.file || isCardType.image || isCardType.user,
-					'card-type-directory form-check form-check-card form-check-middle-left':
-						selectable && horizontal,
-					'file-card': isCardType.file,
-					'form-check form-check-card form-check-top-left':
-						selectable &&
-						(isCardType.file ||
-							isCardType.image ||
-							isCardType.user),
-					'image-card': isCardType.image,
-					'template-card': interactive && !horizontal,
-					'template-card-horizontal': horizontal && interactive,
-					'user-card': isCardType.user,
-				})}
-				href={interactive ? href : undefined}
-				onClick={onClick}
-				role={onClick ? 'button' : undefined}
 			>
-				{(selectable && !horizontal) ||
-				(selectable && isCardType.image) ||
-				isCardType.user ? (
-					<TagName className="card">{children}</TagName>
-				) : (
-					<>{children}</>
-				)}
-			</TagHeaderName>
+				{selectable ? <div className="card">{children}</div> : children}
+			</div>
 		</Context.Provider>
 	);
 };
 
-ClayCard.AspectRatio = AspectRatio;
-ClayCard.Body = Body;
-ClayCard.Caption = Caption;
-ClayCard.Description = Description;
-ClayCard.Group = Group;
-ClayCard.Row = Row;
+const ClayCardHybrid: React.FunctionComponent<IProps> & {
```

Comment:
This component is created only to group all the behaviors that ClayCard currently has but that were separated into other components, in order to maintain compatibility. In the next major version, the idea is to remove and deprecated these behaviors and dry the ClayCard component low-level.

Reply:
I know naming is hard, and maybe `ClayCardHybrid` makes sense, but if we can come up with another name I think it might be worth it

Reply:
@julien yeah, I really can't find another name for it but I would have no problem the idea of it is just to be internal and we removed it soon, also why I changed the `displayName` to `ClayCard` so that it is not visible outside.

Reply:
`Hybrid` should be fine and remains consistent with our naming in Label and Modal